README.rst

```diff
@@ -63,13 +64,10 @@ Additional information for TPROXY
 ---------------------------------
 TPROXY is the only method that supports full support of IPv6 and UDP.
 
-Full UDP or DNS support with the TPROXY method requires the `recvmsg()`
+Full UDP or DNS support with the TPROXY method requires the ``recvmsg()``
 syscall. This is not available in Python 2.7, however is in Python 3.5 and
 later.
```

Comment:
This is the paragraph I was talking about above. Only need PyXAPI for this. Actually I haven't tested PyXAPI for sometime, so might be worth just dropping all references to it - now that we know Python 3.5 has all the support required.